CONTRIBUTING/first-time-contributions.md

```diff
@@ -75,13 +76,3 @@ You can reach us via our [Mailing List](mailto:cncf-wg-env-sustainability@lists.
 
 You can also reach out on our slack [#wg-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6).
 
-### You can reach out to our members
```

Comment:
Why are you removing the member list?

Reply:
We didn't define when is someone a member, yet it doesn't have any value. Is everyone a member who is at the meeting? Or who PRed something? Or wrote something to the mailing list? 

All unclear, therefore removed.

Reply:
We really should consider a place where people can themselves as members.  It was really a welcoming feeling with sig-security when I was able to do so.